sdk/batch/Azure.Compute.Batch/src/Azure.Compute.Batch.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
These should be updated to better reflect what we'd like developers to see on NuGet.

- [Example: Service Bus](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/src/Azure.Messaging.ServiceBus.csproj#L3)

Reply:
update description